### PR TITLE
Improve invite error messages in InviteDialog for room invites

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/15051
Requires https://github.com/matrix-org/matrix-js-sdk/pull/1739

image

### CODE REVIEW

**Comment by turt2live** [comment]
Your CI is unhappy.

**Comment by t3chguy** [comment]
Arg, the typescript return types are incorrect :(

**Comment by turt2live** [comment]
this also probably needs design review? This is reverting the error message behaviour from last time.

File: src/components/structures/MatrixChat.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ import createRoom, {IOpts} from "../../createRoom";
 import {_t, _td, getCurrentLanguage} from '../../languageHandler';
 import SettingsStore from "../../settings/SettingsStore";
 import ThemeController from "../../settings/controllers/ThemeController";
-import { startAnyRegistrationFlow } from "../../Registration.js";
+import { startAnyRegistrationFlow } from "../../Registration";
```
Comment (turt2live) [inline src/components/structures/MatrixChat.tsx:51]: I'm surprised we did this... multiple times.

**Review by turt2live — APPROVED** [review]

**Review by niquewoodhouse — APPROVED** [review]
👍

**Pull request merged**